overlord/fdestate/backend/reseal.go

```diff
@@ -200,7 +191,7 @@ func resealKeys(
 		pbcJSON, _ := json.Marshal(pbc)
 		logger.Debugf("resealing (%d) to boot chains: %s", nextCount, pbcJSON)
 
-		err := resealRunObjectKeys(updateState, runOnlyPbc, pbc, inputs.signatureDBUpdate, authKeyFile, params.RoleToBlName)
+		err := resealRunObjectKeys(manager, runOnlyPbc, pbc, inputs.signatureDBUpdate, params.RoleToBlName)
```

Comment:
it seems these function also needs a new name, like updateRunObjectKeysProtectionProfile or something like that?